CHANGELOG.md

```diff
@@ -107,6 +107,25 @@ Note: This is the first release of Semantic Conventions separate from the Specif
   ([#23](https://github.com/open-telemetry/semantic-conventions/pull/23))
 - Add YAML definitions for log semantic conventions and define requirement levels
   ([#133](https://github.com/open-telemetry/semantic-conventions/pull/133))
+  BREAKING: Add namespaces to JVM metric attributes ([#20](https://github.com/open-telemetry/semantic-conventions/pull/20)).
+ - Rename attributes `type` to `jvm.memory.type`, `pool` to `jvm.memory.pool.name`
+   - Applies to metrics:
+     - `process.runtime.jvm.memory.usage`
+     - `process.runtime.jvm.memory.init`
+     - `process.runtime.jvm.memory.committed`
+     - `process.runtime.jvm.memory.limit`
+     - `process.runtime.jvm.memory.usage_after_last_gc`
+ - Rename attributes `gc`, `action` to `jvm.gc.name`, `jvm.gc.action`
```

Comment:
Will we do a similar thing for other languages? Do we plan to allow every language to put at the top-level of the namespace a concept that applies only to that language?

I would want to understand what is our general strategy with metric attribute namespacing.

Reply:
> Do we plan to allow every language to put at the top-level of the namespace a concept that applies only to that language?

this PR tries to avoid exactly that.
The only attribute we put at the top level is one where we thought it would not be java specific `thread.daemon` (boolean).

Reply:
I am confused. This PR suggests `jvm.` as a top level namespace, which is specific to Java and other JVM languages. How does this PR avoid other languages doing the same and polluting the top-level namespace? Should golang use `go.gc.` namespace for GC metrics?

Reply:
Ignore my previous comment - I think I get it now :smile: 

Yes, this PR suggests that other languages should also be added to the root namespace like `jvm.`. 

I'm not sure what you mean by "polluting" the root namespace - are you suggesting something like `languages.jvm.memory.type`?
For background : The discussion in the Java SIG was that we felt that `jvm.memory.type` was already too long for practical use - but mandated by https://github.com/open-telemetry/semantic-conventions/issues/51.

Reply:
OK, going back to my original question. 

What is our general strategy with regards to choosing top-level namespaces? Is every language free to put anything in the top-level namespace (like Java does with 'jvm.`)? My concern is that this is going to result in highly crowded top-level that is going to be hard to manage. How do we make sure languages don't step on each other's toes? A language can choose to use a namespace and later it turns out to be also needed for another language but can mean something different.

I would like to understand what the policy is in general and I don't see it mentioned anywhere.

>are you suggesting something like languages.jvm.memory.type?

No, I am not suggesting anything. I am merely asking a question.

Reply:
resolving, moved this discussion to #227